api/client/network.go

```diff
@@ -222,7 +224,7 @@ func consolidateIpam(subnets, ranges, gateways []string, auxaddrs map[string]str
 				return nil, fmt.Errorf("multiple overlapping subnet configuration is not supported")
 			}
 		}
-		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}}
+		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}, Options: options}
```

Comment:
On the other hand, it also looks a UI limitation, where backend can manage per-subnet ipam options, while UI cannot provide that in a clean way yet.

We could leave things as is and pass the same options for each subnets to backend. 
If/when UI can provide per subnet separation of options, then we can pass proper option for each subnets with no backend changes.